subsys/bluetooth/host/hci_core.c

```diff
@@ -469,6 +422,55 @@ static void hci_num_completed_packets(struct net_buf *buf)
 		bt_conn_unref(conn);
 	}
 }
+#endif /* CONFIG_BT_CONN || CONFIG_BT_ISO */
+
+#if defined(CONFIG_BT_CONN)
+static void hci_acl(struct net_buf *buf)
```

Comment:
I actually moved the `hci_num_completed_packets` function, but I guess git feels this was a better way to show that